corehq/apps/reports/templates/reports/form/partials/single_form.html

```diff
@@ -170,8 +170,13 @@
         <br/>
         <ul class="list-inline">
             {% if instance.app_id %}<li><a href="{% url "view_app" domain instance.app_id %}">{% trans "View app" %}</a></li>{% endif %}
-            {% if instance.build_id and request.user.is_superuser %}
-                <a href="{% url "download_index" domain instance.build_id %}">{% trans "View build" %}</a>
+            {% if request.user.is_superuser %}
```

Comment:
@proteusvacuum curious why this was a superuser flag and not e.g. a support toggle (or a new one). I thought we were generally phasing out the use of superuser status for enabling/disabling features, but maybe that's no longer the case.

Reply:
No reason other than the "view source files" is under a superuser toggle. Happy to change if you think I should use something else. 

https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/app_manager/templates/app_manager/partials/releases_deploy_modal.html#L338-L354

Reply:
no strong feelings/ not worth changing on my account, was just curious.

Reply:
thinking about this a bit more I wonder if we should create a `solutions` flag for future work? 

it seems like we'll want to decouple solutions work from the ability to access all project spaces at some point, no?